Dockerfile

```diff
@@ -1,10 +1,17 @@
-FROM resin/rpi-raspbian:jessie
-RUN apt-get update
-RUN apt-get install -y curl
-RUN curl -sL https://deb.nodesource.com/setup | bash -
-RUN apt-get install -y build-essential nodejs mongodb
+FROM resin/raspberrypi-node:5.3.0-slim-20160114
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+		python \
+		build-essential \
+		mongodb \
+	&& rm -rf /var/lib/apt/lists/*
+
 COPY . .
+
 RUN mkdir -p /datadb
+
 RUN npm install
```

Comment:
Can you add the `COPY package.json .` step for improved caching of the npm install? :)